libraries/core/src/main/java/com/paritytrading/philadelphia/FIXMessage.java

```diff
@@ -88,6 +88,23 @@ public FIXValue findField(int tag) {
         return null;
     }
 
+    /**
+     * Get the index of the value container of the first instance of a field with the
+     * specified tag.
+     *
+     * @param tag the tag
+     * @return the index or -1 if there are no instances
+     *   of a field with the specified tag
+     */
+    public int findIndex(int tag) {
```

Comment:
Maybe just: "Get the index of the first instance of a field with the specified tag." What do you think? 🤔

Reply:
Yes bit clumsy otherwise. 👍